File: Company-gRPC-service/src/main/java/grpc/project/companygrpcservice/Employee/Employee.java
package grpc.project.companygrpcservice.Employee;

import grpc.project.companygrpcservice.Department.Department;
import grpc.project.companygrpcservice.Person.Person;
import grpc.project.companygrpcservice.Project.Project;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Collection;
import java.util.Date;
@Data @AllArgsConstructor @NoArgsConstructor
public class Employee extends Person {
    private Date doj;
    private Collection<Department> departments;
    private Collection<Project> projects;
}

File: Employee-gRPC-service/Models/Employee/Employee.cs
using Employee_gRPC_service.Enums;

namespace Employee_gRPC_service.Models {
    public class Employee {
        private string? id{get; set;}
        private string? name{get; set;}
        private string? address{get; set;}
        private Gender gender{get; set;}
        private DateTime dob{get; set;}
        private DateTime doj{get; set;}
        private ICollection<Department>? departments{get; set;}
        private ICollection<Project>? projects{get; set;}
    }
}